visualization/app/codeCharta/ui/codeMap/rendering/boxGeometryGenerationHelper.ts

```diff
@@ -138,6 +138,18 @@ export class BoxGeometryGenerationHelper {
 		}
 	}
 
+	private static isTopSide(side: number, node: Node) {
```

Comment:
I guess this function returns an array to indicate which vertice is on a top side. 
What do you think about another name like `determineTopSideVertices`